stellar-node/Docker/init-data/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM stellar/quickstart:latest
+
+COPY init-data.sh /tmp/
```

Comment:
Is this trailing slash needed?

Reply:
Yes, it is a good practice to specify that /, tmp is a directory not a file.